README.rst

```diff
@@ -246,8 +246,8 @@ All config options can be overruled by environment variables. Prefix them with
 options. It is also possible to override logging by setting "HOWITZ_LOGGING" to
 a string of a Python dict but we do not recommend it, use a config file instead.
 
-The refresh interval for the events table can be changed by adding for example ``refresh_interval = 10`` to
-the ``[howitz]``-section or setting the environment variable ``HOWITZ_REFRESH_INTERVAL`` to a new value.
+The refresh interval value specifies the frequency with which Howitz checks the Zino server for updates on events in the table. This value can be modified by adding, for example, ``refresh_interval = 10`` to
```

Comment:
I think this wording will be even more easily misunderstood than the original.

NTIE-based refresh doesn't really *check* with the Zino server.  The Howitz backend has an open notification channel to Zino, so what's really happening here is just that the loaded webpage refreshes from the Howitz server, which returns any changes received from the Zino notification channel.

Maybe something like this?

```suggestion
The refresh interval value specifies the frequency with which the Howitz web page asks the Howitz web server whether the Zino server has sent notifications about updated events. This value can be modified by adding, for example, ``refresh_interval = 10`` to
```

Reply:
Of course, this word salad could keep being mixed forever - it's hard to explain without giving a full primer on the Zino architecture :laughing:

Reply:
fine by me - let's get it out there!